### ISSUE TITLE
SyntaxError: Unexpected token 'export' v 3.0.2

### ISSUE THREAD

**Issue by 3axap4eHko** [apollographql/apollo-client#6666]
**Intended outcome:**
```
import { ApolloLink } from '@apollo/client/link/core';
```

**Actual outcome:**
```
/node_modules/@apollo/client/link/core/index.js:1
export { empty } from './empty';
^^^^^^
SyntaxError: Unexpected token 'export'
```

**Versions**
```
  System:
    OS: Linux 4.19 Ubuntu 18.04.4 LTS (Bionic Beaver)
  Binaries:
    Node: 14.4.0 - ~/.nvm/versions/node/v14.4.0/bin/node
    Yarn: 1.22.4 - /mnt/c/Program Files (x86)/Yarn/bin/yarn
    npm: 6.14.5 - ~/.nvm/versions/node/v14.4.0/bin/npm
  npmPackages:
    @apollo/client: ^3.0.2 => 3.0.2 
    @apollo/react-hooks: ^4.0.0 => 4.0.0 
    apollo-link-persisted-queries: ^0.2.2 => 0.2.2 
    react-apollo: ^3.1.5 => 3.1.5
```

**Comment by benjamn** [comment]
`@apollo/client/link/core` is not a public entry point in 3.0.2 (because it doesn't have a `package.json` file), but it will soon become one in `@apollo/client@3.1.0`, thanks to #6656.

To explain the error, Node.js is trying to import `@apollo/client/link/core/index.js` (which is ESM rather than CommonJS) but fails because Node doesn't natively understand ESM `import`/`export` syntax yet.

**Comment by benjamn** [comment]
Heads up: we're not quite ready to publish `@apollo/client@3.1.0` yet, but I've published an `@apollo/client@3.1.0-pre.0` release that I believe will resolve this issue, if you want to try that in the meantime. 🙏

**Comment by hwillson** [comment]
Let us know if this is still a concern with `@apollo/client@latest` - thanks!
